elrond-node.md

```diff
@@ -105,15 +114,53 @@ Storage maps byte arrays to byte arrays.
                            | "SimulateFailed"           [klabel(SimulateFailed), symbol]
 
     syntax VMOutput ::= ".VMOutput"  [klabel(.VMOutput), symbol]
-                      | VMOutput( returnCode: ReturnCode , returnMessage: Bytes , out: ListBytes, logs: List )
+                      | VMOutput( returnCode: ReturnCode , returnMessage: Bytes , out: ListBytes, logs: List, outputAccounts: Map )
                         [klabel(VMOutput), symbol]
 
- // ------------------------------------------------------------------
+    syntax OutputAccount ::= OutputAccount ( addr: Bytes, transfers: List )             [klabel(OutputAccount), symbol]
+    syntax OutputTransfer ::= OutputTransfer ( sender: Bytes, value: TransferValue )    [klabel(OutputTransfer), symbol]
+
+    syntax Bool ::= nonZeroOutputTransfer(OutputTransfer)      [function, total]
+ // ---------------------------------------------------------------------------
+    rule nonZeroOutputTransfer(OutputTransfer(_, I:Int))                 => I =/=Int 0
+    rule nonZeroOutputTransfer(OutputTransfer(_, esdtTransfer(_, I, _))) => I =/=Int 0
```

Comment:
I'm curious: would it be possible to have `I <Int 0`? Either in a test, or in a real world example.

Reply:
These values are parsed from buffers as unsigned ints, so it is not possible.